doc/paradigmi-per-la-progettazione-delle-interfacce-di-servizio/migrazione-dei-servizi-spcoop-nel-nuovo-framework-di-interoperabilità.rst

```diff
@@ -0,0 +1,38 @@
+Migrazione dei servizi SPCoop nel nuovo framework di interoperabilità
+=====================================================================
+
+Il principio API-first, richiamato nel Piano Triennale per l'informatica
+nella Pubblica Amministrazione, richiede la progettazione dei sistemi
+con un approccio strutturato partendo dalla definizione delle interfacce
+tra le varie componenti. Tutto questo senza dare priorità alle
+tecnologie usate per la loro implementazione (nel caso dei servizi le
+due tecnologie SOAP/REST). In questa logica per i servizi in produzione
+nel contesto del precedente modello di interoperabilità (SPCoop),
+sviluppati in tecnologia SOAP e definiti tramite accordi di servizio e
+pertanto pienamente aderenti al paradigma API First, non viene richiesta
+alcun adeguamento tecnologico. L’unico passo richiesto è quello
+dell'eliminazione del nodo di middleware rappresentato dalla *Porta di
+dominio* (PdD) orientato alla gestione dell’header Intestazione che
+caratterizzava la busta di egov.
+
+L’eliminazione della PdD deve portare l’amministrazione a preservare le
```

Comment:
La decisione credo richieda sempre una valutazione. Una PA potrebbe decidere di ridisegnare un servizio in EOL cogliendo l'occasione della transizione.